components/region_cache_memory_engine/src/background.rs

```diff
@@ -255,16 +262,16 @@ impl BackgroundRunnerCore {
     /// Returns the first range to load with RocksDB snapshot.
     ///
     /// Returns `None` if there are no ranges to load.
-    fn get_range_to_load(&self) -> Option<(CacheRange, Arc<RocksSnapshot>)> {
+    fn get_range_to_load(&self) -> Option<(CacheRange, Arc<RocksSnapshot>, bool)> {
```

Comment:
Please add the comment on the return value `bool`

Reply:
done